src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig

```diff
@@ -42,14 +42,14 @@
         {{ module.attributes.displayName }}
         <small class="text-muted">v{{ module.attributes.version }} - by {{ module.attributes.author }}</small>
         {% if module.attributes.type is not empty and module.attributes.type == 'addonsBought' %}
-        - <span class="module-badge-bought help-tooltip text-warning" data-title="{{ 'You bought this module on PrestaShop Addons. Thank You.'|trans({}) }}"><i class="icon-pushpin"></i> <small>{{ 'Bought'|trans({}) }}</small></span>
+        - <span class="module-badge-bought help-tooltip text-warning" data-title="{{ 'You bought this module on PrestaShop Addons. Thank You.'|trans({}, 'Admin.Modules.Feature') }}"><i class="icon-pushpin"></i> <small>{{ 'Bought'|trans({}, 'Admin.Modules.Feature') }}</small></span>
```

Comment:
Admin.Modules.Help

(but we should comment all this, I think it's no longer used)